### ISSUE TITLE
Error: MalformedPolicyDocumentException

### ISSUE THREAD

**Issue by andrewmackett** [cloudposse/terraform-aws-vpc-flow-logs-s3-bucket#3]
When using this module with the default variable values I receive the error:

```
Error: MalformedPolicyDocumentException: The new key policy will not allow you to update the key policy in the future.
```

I'm using Terraform v0.12.13 with the following providers:
- `template` (hashicorp/template) 2.1.2
- `null` (hashicorp/null) 2.1.2
- `aws` (hashicorp/aws) 2.39.0
- `local` (hashicorp/local) 1.4.0

**Comment by andrewmackett** [comment]
PR #4 allows access from the AWS account to prevent this error occurring.

**Comment by maximmi** [comment]
closed in #4
